### ISSUE TITLE
Shizuku support?

### ISSUE THREAD

**Issue by rohitghali** [Alcatraz323/audiohq_md2#8]
Possible to implement shizuku support for devices without root access?

**Comment by aroguecoder** [comment]
nope, the app hooks itself into a .so on the magisk module that replace the native android lib, you need full root access to do that.
i don't think you can do that with adb or even shizuku.
